src/XRP/network-clients/web-socket-network-client.ts

```diff
@@ -1,19 +1,21 @@
 import WebSocket = require('ws')
 import { XrpError, XrpErrorType } from '../shared'
 import {
-  AccountLinesResponse,
-  GatewayBalancesResponse,
-  WebSocketRequest,
-  StatusResponse,
-  TransactionResponse,
   WebSocketReadyState,
   RippledMethod,
-  ResponseStatus,
+  WebSocketRequest,
   SubscribeRequest,
   AccountLinesRequest,
   GatewayBalancesRequest,
+  AccountOffersRequest,
   WebSocketResponse,
   WebSocketFailureResponse,
+  ResponseStatus,
+  AccountLinesResponse,
+  GatewayBalancesResponse,
+  StatusResponse,
+  TransactionResponse,
+  AccountOffersResponse,
```

Comment:
why did the order of these imports change? it doesn't matter but it will mess up commit history. just curious.

Reply:
see above comment